sys/linux/bpf.txt

```diff
@@ -97,11 +97,30 @@ bpf_prog {
 	prog_name		array[const[0, int8], BPF_OBJ_NAME_LEN]
 	prog_ifindex		ifindex[opt]
 	expected_attach_type	flags[bpf_attach_type, int32]
+	btf_fd			fd_btf
```

Comment:
Probably need to extend bpf_map_create_arg later too, I see it now also contains a btf_fd.